Sming/include/user_config.h

```diff
@@ -6,7 +6,7 @@ extern "C" {
 #endif
 
 	// UART config
-	#define SERIAL_BAUD_RATE 115200
+	#define SERIAL_BAUD_RATE COM_SPEED_SERIAL
```

Comment:
* Where is COM_SPEED_SERIAL defined? 
* Is `921600` good default speed? I would say no, because it is not supported by most of the serial adapters (http://digital.ni.com/public.nsf/allkb/D37754FFA24F7C3F86256706005B9BE7).  I would leave it to  `115200` which is the fastest `supported` speed by most of the devices.